Source/GUI/MemWatcher/Dialogs/DlgAddWatchEntry.cpp

```diff
@@ -33,10 +37,11 @@ void DlgAddWatchEntry::initialiseWidgets()
   connect(m_chkBoundToPointer, &QCheckBox::stateChanged, this,
           &DlgAddWatchEntry::onIsPointerChanged);
 
-  m_lblValuePreview = new QLabel("", this);
+  m_lblValuePreview = new QLineEdit("", this);
+  m_lblValuePreview->setReadOnly(true);
 
   m_txbAddress = new QLineEdit(this);
-  m_txbAddress->setMaxLength(8);
+  m_txbAddress->setMaxLength(10);
```

Comment:
I'm not following why this is 10 now instead of 8? Wouldnt the address always be XXXXYYYY ?

Reply:
It's mentioned in the commit: to allow for the `0x` prefix.

Reply:
Thanks for clarifying.